labman/gui/templates/library_pooling.html

```diff
@@ -0,0 +1,276 @@
+{% extends sitebase.html %}
+
+{% block head %}
+<script type='text/javascript'>
+
+  var poolParams = {
+    'min': [{'prefix': 'floor-vol-', 'value': '100', 'desc': 'volume for low conc samples (nL):', 'min': '1', 'step': '1'},
+            {'prefix': 'floor-conc-', 'value': '20', 'desc': 'minimum value for pooling at real estimated value (nM):', 'min': '0.1', 'step': '0.1'},
+            {'prefix': 'total-nm-', 'value': '0.002', 'desc': 'total number of nM to have in pool (nM):', 'min': '0.00001', 'step': '0.00001'},
+            {'prefix': 'lib-size-', 'value': '500', 'desc': 'Average library molecule size (bp):', 'min': '1', 'step': '1'}],
+    'equal': [{'prefix': 'volume-', 'value': '200', 'desc': 'volume to pool per sample (nL):', 'min': '1', 'step': '1'},
+              {'prefix': 'lib-size-', 'value': '500', 'desc': 'Average library molecule size (bp):', 'min': '1', 'step': '1'}],
+    'floor': [{'prefix': 'min-conc-', 'value': '10', 'desc': 'Minimum concentration to be included in the pool (nM):', 'min': '1', 'step': '1'},
+              {'prefix': 'floor-conc-', 'value': '50', 'desc': 'Minimum value for pooling for samples above min conc (nM):', 'min': '1', 'step': '1'},
+              {'prefix': 'total-nm-', 'value': '0.002', 'desc': 'total number of nM to have in pool (nM):', 'min': '0.00001', 'step': '0.00001'},
+              {'prefix': 'lib-size-', 'value': '500', 'desc': 'Average library molecule size (bp):', 'min': '1', 'step': '1'}],
+    'amplicon': [{'prefix': 'dna-amount-', 'value': '240', 'desc': 'Total amount of DNA (ng):', 'min': '1', 'step': '1'},
+                 {'prefix': 'min-val-', 'value': '1', 'desc': 'Minimum concentration value (ng/&mu;l):', 'min': '0.001', 'step': '0.001'},
+                 {'prefix': 'max-val-', 'value': '15', 'desc': 'Maximum concentration value (ng/&mu;l):', 'min': '0.001', 'step': '0.001'},
+                 {'prefix': 'blank-val-', 'value': '2', 'desc': 'Blanks value:', 'min': '0.001', 'step': '0.001'}]};
+
+  function getPlateInformation(elem) {
+    var plateId = $(elem).attr('pm-data-plate-id');
+    var plateType = $('#plate-type-select').val();
+    var parameterName, poolFunc;
+
+    if (plateType === '16S library prep') {
```

Comment:
how difficult will it be to add in 18S and ITS preparations into labman? is it obvious what all of the changes necessary are?

Reply:
18S and ITS preparations are already supported - The interface exposes everything as 'Amplicon' instead of 16S, but the entire DB and backend uses '16S' instead of amplicon. Fixing this should be simple, but is tedious since that nomenclature is used everywhere in the system. Since that is not affecting critical functionality, that name change has been deferred for the future. Unfortunately the name error was caught when the project was already really advanced.

Reply:
Okay, it might be useful to issue this refactor earlier rather than later in the project as it will only get harder do

Reply:
Agree - added an issue that I've assigned to myself: https://github.com/jdereus/labman/issues/91

Happy to address once the minimum functionality required for publication is in place.